x-pack/plugin/ilm/src/test/java/org/elasticsearch/xpack/indexlifecycle/IndexLifecycleRunnerTests.java

```diff
@@ -1267,7 +1268,7 @@ private void assertClusterStateOnErrorStep(ClusterState oldClusterState, Index i
         assertEquals(currentStep.getAction(), newLifecycleState.getAction());
         assertEquals(ErrorStep.NAME, newLifecycleState.getStep());
         assertEquals(currentStep.getName(), newLifecycleState.getFailedStep());
-        assertEquals(expectedCauseValue, newLifecycleState.getStepInfo());
+        assertThat(newLifecycleState.getStepInfo(), containsString(expectedCauseValue));
```

Comment:
In these tests can we check that the stack trace field is returned so we know we don't break returning the stack trace?

Reply:
Certainly